src/utils.test.js

```diff
@@ -0,0 +1,63 @@
+import 'jest';
+import { getDataHubParams } from './utils';
+
+describe('utils', () => {
+  describe('getDataHubParams', () => {
+    it('should parse url with no params and no hash correctly', () => {
+      //Arrange
+      //Act
+      const dataHubParams = getDataHubParams('/login');
+      //Assert
+      expect(dataHubParams.navigatedPage).toBe('/login');
+      expect(dataHubParams.search).toBe('');
+      expect(dataHubParams.hash).toBe('');
```

Comment:
Is it fine? or it should be null? @leoslopez what do you think?

Reply:
It would be the same for the Datahub [interpretation](https://github.com/MakingSense/Customer-Data-Hub/blob/develop/public/dhapi.js#L117-L119): 
![image](https://user-images.githubusercontent.com/1985175/61550994-60bf2a00-aa2a-11e9-89f1-1d66cad6a990.png)

It means that, in case that the values are missing (null or empty too) in options, the values will be obtained from url.

But this makes me doubt about the proper interpretation of Datahub.
Maybe Datahub should do:
```
if (enableCustomTracking) {
  // consider values received
} else {
  // consider values from window.location
}
```
What do you think?

Reply:
Oh, you are right, good catch @leoslopez !

I like the change you are proposing.